handlers/token_balance_api.py

```diff
@@ -0,0 +1,63 @@
+import json
+from typing import Optional
+
+from aws_lambda_context import LambdaContext
+
+from common.errors import ApiError
+from usecases.token_balances_api import TokenBalancesApi
+from utils.wrappers.aws.api_gateway import ApiGateway, ApiGatewayEvent
+
+
+@ApiGateway()
+def get_token_balances(
+    event: ApiGatewayEvent,
+    _context: LambdaContext,
+    token_balances_api: Optional[TokenBalancesApi] = None
+) -> dict:
+    """Get token balances from user search"""
+
+    token_balances_api = token_balances_api or TokenBalancesApi()
+
+    token_id = event.query.get('token_id') or "00"
```

Comment:
Maybe we could have a constant to keep this `00`, like `HTR_TOKEN_ID = 00` (it could be in this same file)

Just to improve reading of the code

Reply:
We could get it from hathor lib, but we don't have it on the project

Using a constant on 2f185863d50dbca7e21d708208018ae48294b105